### PR TITLE
OBVSG

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

**Comment by zuphilip** [comment]
@kba What is the problem with Travis?

**Comment by stweil** [comment]
Travis did its job and failed because of an error in the new code. A closing `'` was missing, see my code comment.

**Comment by stweil** [comment]
@zuphilip, I fixed the closing `'`. Now Travis reports a whitespace at end of line in file `/home/travis/build/UB-Mannheim/malibu/isbn/lib.php`.

**Comment by zuphilip** [comment]
Okay, I see. Shall I take it from here?

File: isbn/lib.php [hunk 1]
```diff
@@ -153,6 +153,8 @@ function cleanUp($outputMap)
     }
     if (is_array($outputMap['rvk'])) {
         foreach ($outputMap['rvk'] as $i => $value) {
+            //e.g. <feld nr="700" ind="g">|SK 830<tf/>Automatisch aus BVB_2013-06 2013-03-27</feld>
```
Comment (stweil) [inline isbn/lib.php]: Whitespace after `</feld>` at end of line detected by Travis. Fixed now.

**Comment by stweil** [comment]
Pulled, thanks.

**Comment by zuphilip** [comment]
Thank you very much @stweil for the help! :tada: I prefer to merge the PR via github, but the result is the same (just more work for travis).

**Comment by zuphilip** [comment]
Actually, it seems that there is a Travis problem now in `master`: https://github.com/UB-Mannheim/malibu/branches

**Comment by stweil** [comment]
That problem in master was caused by the whitespace issue. It should be fixed now.

**Comment by zuphilip** [comment]
Yes, it is fixed now.

**Pull request closed without merging**